task/ike-session-url/0.1/ike-session-url.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: ike-session-url
+  labels:
+    app.kubernetes.io/version: 0.1
```

Comment:
```suggestion
    app.kubernetes.io/version: "0.1"
```

Reply:
@chmouel I guess that's another reason why semver would be less painful to deal with ;)